server/src/main/java/com/strategists/game/controller/UpdateController.java

```diff
@@ -11,6 +12,7 @@
 
 @RestController
 @RequestMapping("/api/updates")
+@CrossOrigin("*")
```

Comment:
We don't need this. Adding proxy in frontend package.json will work

https://create-react-app.dev/docs/proxying-api-requests-in-development/

Reply:
Adding it to `package.json` only works for development purposes. It wouldn't work for the production use-case.

EventSource API doesn't respect React's proxy.